pkg/test/framework/components/istio/config.go

```diff
@@ -127,6 +128,10 @@ type Config struct {
 	// CustomSidecarInjectorNamespace allows injecting the sidecar from the specified namespace.
 	// if the value is "", use the default sidecar injection instead.
 	CustomSidecarInjectorNamespace string
+
+	//If false,  all remote clusters will be the new istiod-less remotes.
+	//If true, the "legacy" remote with istiod will be used.
+	UseLegacyRemote bool
```

Comment:
Let's invert this so that it has a sensible default (`false`). Also most checks use `!UseLegacyRemote`. How about we call this something like `UseLightweightRemoteClusters`, and default to `false`?

Reply:
agree with this!